### PR TITLE
Csptoken check the claim item is null for compatible all clients token

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Review by jin-jasper — APPROVED** [review]

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java [hunk 1]
```diff
@@ -130,11 +130,19 @@ public void verify(JWTClaimsSet claimsSet, SecurityContext c) throws BadJWTExcep
         claim.setSub(claimSet.getClaim("sub").toString());
         claim.setExp(claimSet.getClaim("exp").toString());
         claim.setIat(claimSet.getClaim("iat").toString());
-        claim.setAcct(claimSet.getClaim("acct").toString());
-        claim.setDomain(claimSet.getClaim("domain").toString());
-        claim.setContext(claimSet.getClaim("context").toString());
         claim.setContextName(claimSet.getClaim("context_name").toString());
-        claim.setPerms(claimSet.getStringArrayClaim("perms"));
+        if (claimSet.getClaim("acct") != null){
```
Comment (huihuiw01) [inline g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java:134]: The bug's requirment is remove 'acct' and 'domain', why not remove?

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/CspValidateService.java [hunk 2]
```diff
@@ -132,11 +132,19 @@ public void verify(JWTClaimsSet claimsSet, SecurityContext c) throws BadJWTExcep
         claim.setSub(claimSet.getClaim("sub").toString());
         claim.setExp(claimSet.getClaim("exp").toString());
         claim.setIat(claimSet.getClaim("iat").toString());
-        claim.setAcct(claimSet.getClaim("acct").toString());
-        claim.setDomain(claimSet.getClaim("domain").toString());
-        claim.setContext(claimSet.getClaim("context").toString());
         claim.setContextName(claimSet.getClaim("context_name").toString());
-        claim.setPerms(claimSet.getStringArrayClaim("perms"));
+        if (claimSet.getClaim("acct") != null){
```
Comment (huihuiw01) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/CspValidateService.java:136]: Same question with above comment

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java:134]: I have add the comments in issue. It needn't remove just check the null case.

Comment (huihuiw01) [reply g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java:134]: ok, please add 'It needn't remove just check the null case.' to your comment on issue to clarify change needed.

**Review by huihuiw01 — APPROVED** [review]

**Pull request merged**